recipe/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "barriers" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/b/barriers/barriers-1.1.0.tar.gz
+  sha256: dee86a3b523277321095405707cee00ce6b9fd40a423c98588e663ba498ed4fc
+
+build:
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - setuptools 67.8.0  # because upstream requires ~=67
```

```suggestion
    - setuptools
```
It's a super limiting constrain. It will works fine with our latest setuptools

---

should we then always ignore the upstream requirements? I wonder why the upstream project even bothered to specify then.

---

Or is it a case where as long as it builds we don't care about the `setuptools` version?

---

> Or is it a case where as long as it builds we don't care about the `setuptools` version?

Yes.

---

yes, it is too restrictive remove that version and the LGTM